xtask/src/tasks.rs

```diff
@@ -121,6 +126,26 @@ pub fn fuzz_corpus_push(sh: &Shell) -> anyhow::Result<()> {
     Ok(())
 }
 
+pub fn fuzz_install(sh: &Shell) -> anyhow::Result<()> {
+    let _s = Section::new("FUZZ-INSTALL");
+
+    let cargo_fuzz_path: std::path::PathBuf = [LOCAL_CARGO_ROOT, "bin", "cargo-fuzz"].iter().collect();
```

Comment:
FYI: I had a similar idea for installing `typos` in the local root, and then caching compiled tool binaries, where the cache key will be the hash of a separate `xtask.toml` config file, containing the required tool versions to be installed. e.g.
```toml
[tool.typos]
version = "1.14.9"

[tool.cargo-fuzz]
version = "0.11.2"
```
xtask will search for the following:
- Search for **globally** installed tool on the machine (>= required_version)
- Search **local_root** for already built(or cached) binaries (>=required_version, additional local_root's binary version check required if developer made branch switch/rebase/etc. and `target` was not cleaned prior to xtask invocation)
- If all of the above failed - **clean** local_root's tool directory and **build** brand new version via `cargo install`

I'll prepare PR this week (maybe next) which will generalise tool installation/cache logic

Reply:
Sounds very good to me! Looking forward to it :+1: